Source/Engine/UI/GUI/RenderOutputControl.cs

```diff
@@ -221,7 +221,7 @@ public override void Draw()
         /// </summary>
         public void SyncBackbufferSize()
         {
-            float scale = ResolutionScale * Platform.DpiScale;
+            float scale = ResolutionScale * (Root?.RootWindow?.Window?.DpiScale ?? 1); // TODO: Figure this out
```

Comment:
Apparently sometimes the `RenderOutputControl`s don't have a `RootWindow`?

Reply:
It might be missing if window is unlinked and this sync gets called but it's ok.